### PR TITLE
8274383 JNI call of getAccessibleSelection on a wrong thread

### PR DESCRIPTION
**Pull request by savoptik**
[ComboBoxAccessibility accessibilityValue] defines the getAccessibleSelection method on the AccessibleContext class, however the call should go through CAccessibility so that it is executed on the Event Dispatch thread.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8274383](https://bugs.openjdk.java.net/browse/JDK-8274383): JNI call of getAccessibleSelection on a wrong thread

### Reviewers
 * [Alexander Zuev](https://openjdk.java.net/census#kizune) (@azuev-java - **Reviewer**) ⚠️ Review applies to 8adcb24fc558d38b982cc046b6fd2cdd02b65ce9
 * [Anton Tarasov](https://openjdk.java.net/census#ant) (@forantar - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/5735/head:pull/5735` \
`$ git checkout pull/5735`

Update a local copy of the PR: \
`$ git checkout pull/5735` \
`$ git pull https://git.openjdk.java.net/jdk pull/5735/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 5735`

View PR using the GUI difftool: \
`$ git pr show -t 5735`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/5735.diff">https://git.openjdk.java.net/jdk/pull/5735.diff</a>

</details>

### CODE REVIEW

File: src/java.desktop/macosx/native/libawt_lwawt/awt/a11y/ComboBoxAccessibility.m [hunk 1]
```diff
@@ -43,15 +49,20 @@ - (id)accessibilityValue {
     JNIEnv *env = [ThreadUtilities getJNIEnv];
     jobject axContext = [self axContextWithEnv:env];
     if (axContext == NULL) return nil;
-    jclass axContextClass = (*env)->GetObjectClass(env, axContext);
-    DECLARE_METHOD_RETURN(jm_getAccessibleSelection, axContextClass, "getAccessibleSelection", "(I)Ljavax/accessibility/Accessible;", nil);
-    jobject axSelectedChild = (*env)->CallObjectMethod(env, axContext, jm_getAccessibleSelection, 0);
+    GET_ACCESSIBLESELECTION_METHOD_RETURN(nil);
+    jobject axSelection = (*env)->CallStaticObjectMethod(env, sjc_CAccessibility, sjm_getAccessibleSelection, axContext, self->fComponent);
```
[the remark this answers is not in the document]
Comment (savoptik) [reply src/java.desktop/macosx/native/libawt_lwawt/awt/a11y/ComboBoxAccessibility.m:53]: Done

**Review by forantar — APPROVED** [review]

**Review by pankaj-bansal — COMMENTED** [review]
Looks good to me

**Pull request closed without merging**